.github/workflows/call-build-windows.yaml

```diff
@@ -61,11 +73,17 @@ jobs:
       - name: Set up Visual Studio shell
         uses: egor-tensin/vs-shell@v2
         with:
-          arch: x64
+          arch: ${{ matrix.config.arch }}
+
+      - name: Build openssl with vcpkg
+        if: ${{ matrix.config.arch == 'x86' }}
+        run: |
+          C:\vcpkg\vcpkg install --recurse openssl --triplet ${{ matrix.config.vcpkg_triplet }}
+        shell: cmd
 
       - name: Build Fluent Bit packages
         run: |
-          cmake -G "NMake Makefiles" -DFLB_NIGHTLY_BUILD=${{ inputs.unstable }} ../
+          cmake -G "NMake Makefiles" -DFLB_NIGHTLY_BUILD=${{ inputs.unstable }} ${{ matrix.config.openssl_root_dir_opt }} ../
           cmake --build .
           cpack
         working-directory: build
```

Comment:
@cosmo0920 can you add the MSI output as well to what we upload?
https://github.com/fluent/fluent-bit/blob/a6a1aa03a8f5c63b67603825771653940192f1fa/.github/workflows/call-build-windows.yaml#L97

Reply:
Actually LGTM and I'll sort that as need to tweak the server sync as well anyway for it.